ui/tests/integration/components/oidc/assignment-form-test.js

```diff
@@ -3,27 +3,23 @@ import { setupRenderingTest } from 'ember-qunit';
 import { render, fillIn, click, findAll } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import { setupMirage } from 'ember-cli-mirage/test-support';
+import ENV from 'vault/config/environment';
 import { overrideMirageResponse } from 'vault/tests/helpers/oidc-config';
 
 module('Integration | Component | oidc/assignment-form', function (hooks) {
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
+  hooks.before(function () {
+    ENV['ember-cli-mirage'].handler = 'oidcConfig';
+  });
+
+  hooks.after(function () {
+    ENV['ember-cli-mirage'].handler = null;
+  });
+
   hooks.beforeEach(function () {
     this.store = this.owner.lookup('service:store');
-    this.server.post('/sys/capabilities-self', () => {});
```

Comment:
pulled this into reusable mirage handler file utilized in both `client-form-test` and here